setup.cfg

```diff
@@ -0,0 +1,65 @@
+[metadata]
+name = hankel
+description = Hankel Transformations using method of Ogata 2005
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/steven-murray/hankel
+author = Steven Murray
+author_email = steven.g.murray@asu.edu
+license = MIT
+license_file = LICENSE.rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Intended Audience :: End Users/Desktop
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering
+    Topic :: Utilities
+project_urls =
+    Documentation = https://hankel.readthedocs.io
+    Source = https://github.com/steven-murray/hankel
+    Tracker = https://github.com/steven-murray/hankel/issues
+    Changelog = https://github.com/steven-murray/hankel/blob/master/CHANGELOG.rst
+    Conda-Forge = https://anaconda.org/conda-forge/hankel
+
+[options]
+packages = hankel
+install_requires =
+    mpmath>=1.0.0
+    numpy>=1.9.3
+    scipy>=0.16.1
+python_requires = >=3.6
+setup_requires =
+    setuptools>=44;wheel; setuptools_scm[toml]>=3.4.3
+
+[options.extras_require]
+dev =
+    ipykernel
+    nbsphinx
+    numpydoc
+    pandoc
+    pre-commit
+    pytest
+    pytest-cov
+    sphinx>=2.0<3
+    sphinx-rtd-theme
+docs =
+    ipykernel
+    nbsphinx
+    numpydoc
+    pandoc
+    sphinx>=2.0<3
+    sphinx-rtd-theme
+tests =
+    pre-commit;pytest; pytest-cov
```

Comment:
Why isn't pytest-cov a test dependency? (pytest would be redundant)

Reply:
Just a bug in the automatic convertor I use.